Sources/ApolloWebSocket/ApolloWebSocket.swift

```diff
@@ -5,48 +5,40 @@ import Foundation
 
 /// Protocol allowing alternative implementations of web sockets beyond `ApolloWebSocket`. Extends `Starscream`'s `WebSocketClient` protocol.
 public protocol ApolloWebSocketClient: WebSocketClient {
-
+  
   /// Required initializer
   ///
-  /// - Parameter request: The URLRequest to use on connection.
-  /// - Parameter protocols: The supported protocols
-  init(request: URLRequest, protocols: [String]?)
+  /// - Parameters:
+  ///   - request: The URLRequest to use on connection.
+  ///   - certPinner: [optional] The object providing information about certificate pinning. Should default to Starscream's `FoundationSecurity`.
+  ///   - compressionHandler: [optional] The object helping with any compression handling. Should default to nil.
+  init(request: URLRequest,
+       certPinner: CertificatePinning?,
+       compressionHandler: CompressionHandler?)
 
   /// The URLRequest used on connection.
   var request: URLRequest { get set }
 
   /// Queue where the callbacks are executed
   var callbackQueue: DispatchQueue { get set }
-}
-
-public protocol SOCKSProxyable {
   
-  /// Determines whether a SOCKS proxy is enabled on the underlying request.
-  /// Mostly useful for debugging with tools like Charles Proxy.
-  var enableSOCKSProxy: Bool { get set }
```

Comment:
unfortunately v4 of Starscream doesn't support proxying at this time, so I had to remove this.